website/www/site/static/js/copy-to-clipboard.js

```diff
@@ -13,15 +13,15 @@
 $(document).ready(function() {
     function copy() {
         $(".copy").click(function(){
-            var text=$(this).siblings()[$(this).siblings().length-1].childNodes[0].innerHTML;
+            var text=$(this).siblings()[$(this).siblings().length-1].childNodes[0].innerText;
             const el=document.createElement('textarea');
             el.value=text;document.body.appendChild(el);
             el.select();document.execCommand('copy');
             document.body.removeChild(el);
             alert('copied to clipboard');
```

Comment:
should we leave this alert here? is there a less intrusive way of noting that things have been copied? the alert requires you to accept, which is kind of annoying...

Reply:
same for the alert on line 29

Reply:
Agree with @pabloem. Do we need this alert at all?

Reply:
Thanks for the feedback @pabloem and @aromanenko-dev !
Based on your inputs, Is it ok if I remove both alerts?

Reply:
+1 to remove it. @pabloem, wdyt?

Reply:
@pabloem I went ahead and submitted the change.
Please let me know if you have further comments!
Thanks again!